_sass/landing/landing.scss

```diff
@@ -11,7 +11,7 @@
     border: $input-border solid $gray-lighter;
     border-radius: $input-border-radius-lg;
     box-shadow: 0 2px 9px rgba($black,.075);
-    margin: 120px auto;
+    margin: 120px auto 60px;
```

Comment:
This selector moved to global.scss. Remove this change, commit, pull latest master, then make this change.